src/components/website-management/website-form-components/customization/customization-components/PopUp.js

```diff
@@ -0,0 +1,158 @@
+import Dialog from "@mui/material/Dialog";
+import DialogActions from "@mui/material/DialogActions";
+import DialogContent from "@mui/material/DialogContent";
+import DialogTitle from "@mui/material/DialogTitle";
+import { useEffect, useState } from "react";
+import { Button, TextField } from "@mui/material";
+import Typography from "@mui/material/Typography";
+import CancelIcon from "@mui/icons-material/Cancel";
+import axios from "axios";
+
+const PopUp = (props) => {
+  const { openPopup, handleClose, data, getTemps } = props;
+
+  const [textBox01, setTextBox01] = useState();
+
+  const [textBox02, setTextBox02] = useState();
+
+  const [textBox03, setTextBox03] = useState();
+
+  const [textBox04, setTextBox04] = useState();
+
+  const template = {
+    templateName: textBox01,
+    templateRegulation: textBox02,
+    templatePrivacyPolicyLink: textBox03,
+    templateContent: textBox04,
+  };
+  const handleSave = async () => {
+    const id = data.templateId;
+    await axios
+      .put(
+        `http://localhost:8080/api/template/updateTemplateID/${id}`,
+        template
+      )
+      .then(() => getTemps());
```

Comment:
Use a library such as [react query](https://tanstack.com/query/latest/docs/react/overview) for data fetching. 

PS: Let's do this as a separate PR, not in this PR.

Reply:
noted.